### ISSUE TITLE
OAI Provider - Jupiter

### ISSUE THREAD

**Issue by weiweishi** [ualbertalib/fcrepo4-oaiprovider#22]
This is an epic to capture the work required for OAI Provider needed to build on top of the OAI provider on Fedora 4.7 community version, with added functionality and customization:

- [ ] based on out of box OAI provider for Fedora 4.7 and add
- [ ] ETDMS, and 
- [ ] ORE
- [ ] Additional customization/enhancement
  - [ ] How are we going to handle definition of sets to support our collections. 
  - [ ] Permission? 
    - Jupiter's permission model? 

Out of scope:
- [ ] Use triplestore for record retrieval.

**Comment by jefferya** [comment]
* update mvn build to 4.7.4
  - [x] update to use 4.7.4 libraries
  - [x] fix breaking changes in codebase: compile time
  - [x] fix stratup errors
 
* update config to 4.7.4 
  - [x] ModeShape config: respostory.json

* data model changes: crosswalk to OAI output
  - [x] namespaces (modify oai.xml and prevent startup errors of newly built fcrepo-data directory due to modeshape repository.json using unknown namespaces via `node-types_ual.cnd`
  - [x] jcr queries relating to hasModel
  - [] memberOf (collection and community)
  - [x] index updates in ModeShape repository json
  - [x] dcterms:accessRights


ToDo:
* test OAI verb `ListSets` once `pcdm:memberOf` is in place to link collections to communities
* test `dcTerms:accessRights` is public
* test `hasModel` options 
* inspect code comments about JCR Workarounds to see if still valid with Fcrepo 4.7.4

**Comment by jefferya** [comment]
superseded by https://github.com/ualbertalib/jupiter/issues/376
